Src/AutoFixture.NUnit2.UnitTest/FrozenAttributeTest.cs

```diff
@@ -31,12 +33,15 @@ public void GetCustomizationFromNullParamterThrows()
         }
 
         [Test]
-        public void GetCustomizationReturnsCorrectResult()

```

Comment:
There may be something that I'm unable to follow here, but I don't understand why this test is being deleted. Isn't this test exercising the default behaviour of the `[Frozen]` attribute, when no properties are set? Shouldn't that behaviour remain as it was before?